scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -920,3 +920,30 @@ def c1(x):
         assert_(np.all(np.array(c1(res.x)) <= np.array([92, 110, 25])))
         assert_(np.all(res.x >= np.array(bounds)[:, 0]))
         assert_(np.all(res.x <= np.array(bounds)[:, 1]))
+
+    def test_L9(self):
+        # Lampinen ([5]) test problem 9
+
+        def f(x):
+            x = np.hstack(([0], x))  # 1-indexed to match reference
+            return x[1]**2 + (x[2]-1)**2
+
+        def c1(x):
+            x = np.hstack(([0], x))  # 1-indexed to match reference
+            return [x[2] - x[1]**2]
+
+        N = NonlinearConstraint(c1, [-.001], [0.001])
+
+        bounds = [(-1, 1)]*2
+        constraints = (N)
+        res = differential_evolution(f, bounds, maxiter=1000, popsize=30,
```

Comment:
As an example for the different population sizes employed - this example will use a total population size of 60 (`len(x) * popsize`). In comparison Lampinen uses a population size of 30 in the paper. All other things being equal this would mean it would take twice as long for each iteration.

Reply:
Ok. That is easily fixed. I've never used `differential_evolution` before this, and I didn't study the difference in terminology between the paper and the options.